README.md

```diff
@@ -3,63 +3,26 @@ Heapster enables monitoring of clusters using [cAdvisor](https://github.com/goog
+### Running Heapster on Kubernetes
 
-3. To access the InfluxDB UI, you will have to open up the InfluxDB UI port (8083) on the nodes. You can do so by creating a firewall rule:
+To run Heapster on a Kubernetes cluster targetting [InfluxDB](http://influxdb.com) and [Grafana](http://grafana.org/docs/features/influxdb) use [this guide](docs/influxdb.md). Alternatively you can also use [Google Cloud Monitoring](https://cloud.google.com/monitoring/) with [this guide](docs/gcm.md).
 
-	```shell
-$ gcloud compute firewall-rules create monitoring-heapster --allow "tcp:8083" "tcp:8086" --target-tags=kubernetes-minion
-	```
+While running on Kubernetes, Heapster does the following:
 
-	Then, find out the IP address of the node where InfluxDB is running and point your web browser to `http://<nodeIP>:8083/`.
-
-	_Note: We are working on exposing the InfluxDB UI using the proxy service on the Kubernetes master._
-
-4. If you find InfluxDB to be using up a lot of CPU or memory, consider placing resource restrictions on the `InfluxDB & Grafana` pod. You can add `cpu: <millicores>` and `memory: <bytes>` in the [Controller Spec](deploy/influxdb-grafana-controller.yaml) and relaunch the controllers:
-
-	```shell
-$ deploy/kube.sh restart
-	```
-
-##### Hints
-* To enable memory and swap accounting on the minions, follow [these instructions](https://docs.docker.com/installation/ubuntulinux/#memory-and-swap-accounting).
-
-#### How Heapster works on Kubernetes
 1. Discovers all minions in a Kubernetes cluster
 2. Collects container statistics from the kubelets running on the nodes
-2. Organizes stats into pods and adds Kubernetes-specific metadata
-3. Stores pod stats in a configurable backend
+3. Organizes stats into pods and adds Kubernetes-specific metadata
+4. Stores pod stats in a configurable backend
 
 Along with each container stat entry, its pod ID, container name, pod IP, hostname and labels are also stored. Labels are stored as `key:value` pairs.
 
-Heapster currently supports [InfluxDB](http://influxdb.com) and [BigQuery](https://cloud.google.com/bigquery/) backends. We welcome patches that add additional storage backends.
+Heapster currently supports [InfluxDB](http://influxdb.com), [Google Cloud Monitoring](https://cloud.google.com/monitoring/), and [BigQuery](https://cloud.google.com/bigquery/) backends. We welcome patches that add additional storage backends.
+
+### Running outside of Kubernetes
+
+Heapster can be used to enable cluster-wide monitoring on other cluster management solutions by running a simple cluster-specific buddy container that will help Heapster with discovery of hosts. For example, take a look at [this guide](clusters/coreos/README.md) for setting up cluster monitoring in [CoreOS](https://coreos.com) without using Kubernetes.
```

Comment:
Coreos support no longer requires the buddy. This section needs to be updated. We can do this in a later PR.